apps/src/code-studio/initApp/project.js

```diff
@@ -1083,6 +1092,18 @@ function fetchAbuseScore(resolve) {
   });
 }
 
+function fetchSharingDisabled(resolve) {
+  channels.fetch(current.id + '/sharing_disabled', function (err, data) {
+    sharingDisabled = (data && data.sharing_disabled) || sharingDisabled;
+    resolve();
+    if (err) {
+      // Throw an error so that things like New Relic see this. This shouldn't
+      // affect anything else
+      throw err;
+    }
+  });
```

Comment:
Some things we might want to change in the future:
- It would probably be better to have this return a promise rather than take in a callback
- If we taking in a callback, as we are here, I'd rather have it named `callback` instead of `resolve`
- It feels particularly gross,  knowing that this is a promise, that we still resolve on errs and then throw instead of rejecting.

That said, it makes sense for now to follow the pattern used in `fetchPrivacyProfanityViolations`  and elsewhere, so probably not worth actually changing anything here right now.